tests/benchdnn/pool/pool.cpp

```diff
@@ -282,6 +292,11 @@ int doit(const prb_t *prb, res_t *res) {
             ref_args.set(DNNL_ARG_DIFF_DST, d_dst_fp);
             ref_args.set(DNNL_ARG_DIFF_SRC, d_src_fp);
 
+#if DNNL_AARCH64_USE_ACL
+            // This is needed for pool::get_initial_min_value()
+            // See https://github.com/oneapi-src/oneDNN/issues/1205
+            const_cast<prb_t *>(prb)->is_acl = (res->impl_name == "acl");
```

Comment:
This looks like a hack, is it possible to properly initialize it?

Reply:
`prb` is initialized in `check_correctness(settings_t)` which then calls `doit()`. But the implementation is picked in `doit()` so it isn't known in `check_correctness(settings_t)`